pkgs/by-name/tt/tts/package.nix.orig

```diff
@@ -0,0 +1,210 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  espeak-ng,
+  tts,
+<<<<<<< HEAD
+  addBinToPathHook,
+  writableTmpDirAsHomeHook,
+=======
+  nix-update-script,
+>>>>>>> f557e19b553b (tts: add generic nix-update-script)
```

Comment:
Looks like a conflict.

Reply:
Looks like this file wasn't meant to be part of the PR at all? Note the `.nix.orig` extension

Reply:
oups, right, sry about that.